### PR TITLE
docs: update tsg

### PR DESCRIPTION
**Pull request by binbin-li**
1. Update TSGs for notation verifier and KMP
2. Add new errors in error list.

### CODE REVIEW

**Comment by susanshi** [comment]
Hi @junczhu, @shahramk64 , would you be able to do a initial review? thanks!

File: docs/troubleshoot/verifier/notation.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
 # Troubleshoot Notation Verifier Errors
 
-If an image verification failed for some reason, please check the logs in Ratify pod to inspect the related error logs.
+There are generally two types of errors that can occur when using the Notation verifier: configuration errors and verification errors. Configuration errors occur when the Notation verifier is not configured correctly. Verification errors occur when the Notation verifier fails to verify an artifact. This TSG provides guidance on how to troubleshoot Notation verifier errors.
```
Comment (shahramk64) [inline docs/troubleshoot/verifier/notation.md:3]: Suggestion: This TSG provides guidance on how to troubleshoot both types of Notation verifier errors.

File: docs/troubleshoot/verifier/notation.md [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
 
-Below is an example of a result log that occurs when Notation verifier fails to verify an image.
+## Configuration Errors
+It's important to ensure that the Notation verifier is configured correctly. If the Notation verifier is not configured correctly, it will not be able to verify artifacts.
```
Comment (shahramk64) [inline docs/troubleshoot/verifier/notation.md:6]: Maybe add this at the end: For more information on how to configure the Notation verifier, please refer to [this guide](https://ratify.dev/docs/plugins/verifier/notation).

File: docs/troubleshoot/verifier/notation.md [hunk 3]
```diff
@@ -23,32 +50,21 @@ Below is an example of a result log that occurs when Notation verifier fails to
...
-
-Since the other levels of the error are always the same, this TSG focuses on different errors returned by Notation verifier.
-
-## Debugging Commands
-To inspect the Notation verifier configuration, please use ```kubectl describe``` or ```kubectl get``` command to retrieve it.
-```bash
-kubectl describe verifiers.config.ratify.deislabs.io
-```
-or
+Users can investigate the root cause of the Notation verifier by checking the `message` and `errorReason` fields of each failed `verifierReport`. `message` is generated within the Ratify while `errorReason` could result from Ratify itself, dependency library or upstream services. And we have `remediation` field to provide a solution to some common errors.
```
Comment (shahramk64) [inline docs/troubleshoot/verifier/notation.md:53]: within ~~the~~ Ratify

**Review by shahramk64 — APPROVED** [review]
LGTM.
Just a few minor comments.

File: docs/troubleshoot/verifier/notation.md [hunk 4]
```diff
@@ -1,8 +1,32 @@
...
-If an image verification failed for some reason, please check the logs in Ratify pod to inspect the related error logs.
+There are generally two types of errors that can occur when using the Notation verifier: configuration errors and verification errors. Configuration errors occur when the Notation verifier is not configured correctly. Verification errors occur when the Notation verifier fails to verify an artifact. This TSG provides guidance on how to troubleshoot both types of Notation verifier errors.
 
-Below is an example of a result log that occurs when Notation verifier fails to verify an image.
+## Configuration Errors
+It's important to ensure that the Notation verifier is configured correctly. If the Notation verifier is not configured correctly, it will not be able to verify artifacts. For more information on how to configure the Notation verifier, please refer to [this guide](https://ratify.dev/docs/plugins/verifier/notation/).
+
+### Debugging Commands
+To inspect the Notation verifier configuration, please use ```kubectl describe``` or ```kubectl get``` command to retrieve it.
+```bash
```
Comment (junczhu) [inline docs/troubleshoot/verifier/notation.md:10]: It's good practice to wrap triple backticks (```) used for fenced code blocks with blank lines before and after the code block. 
This helps in ensuring that the Markdown renderer correctly identifies the code block and maintains readability in the raw Markdown file.

Comment (binbin-li) [reply docs/troubleshoot/verifier/notation.md:10]: good call!

**Review by susanshi — APPROVED** [review]
LGTM. thanks Binbin!

**Pull request merged**